scripts/apps/desks/directives/DeskeditBasic.js

```diff
@@ -93,6 +95,10 @@ export function DeskeditBasic(gettext, desks, WizardHandler, metadata, $filter,
                 }
             };
 
+            scope.showNoPublishOnAuthoringDesk = function(deskType) {
```

Comment:
In your other PR #1850 you've called it `canPublishOnDesk`. Which is a much better name. Please use the same here for consistency.

Reply:
Here the meaning is to show the information about "NoPublishOnAuthoringDesk"